core/commands/swarm.go

```diff
@@ -130,6 +134,30 @@ var swarmPeeringAddCmd = &cmds.Command{
 				return err
 			}
 		}
+
+		save, _ := req.Options[swarmSaveOptionName].(bool)
```

Comment:
can we check here if the casting is not ok to throw a descriptive error to the user?

Reply:
@ajnavarro no this is already enforced by `cmds.BoolOption`, also the `save, _` will just put `save` to `false` if it's not a bool.

Reply:
@Jorropo exactly, I'm trying to avoid headaches to users that set `"true"` instead of `true`